cern_access/indico_cern_access/models/access_requests.py

```diff
@@ -39,7 +39,7 @@ class CERNAccessRequest(db.Model):
     )
     reservation_code = db.Column(
         db.String,
-        nullable=False
```

Comment:
I'd rather be explicit and put `nullable=True` here instead of falling back to the implicit default